deftest/mock/mock.lua

```diff
@@ -92,10 +92,8 @@ function mock.mock(module)
 					mock_fn.calls = mock_fn.calls + 1
 					local arg = {...}
 
-					if #arg > 0 then
-						for i=1,#arg do
-							mock_fn.params[i] = arg[i]
-						end
+					for index, value in pairs(arg) do
```

Comment:
Hmm, perhaps we need to make sure that mock_fn.params is cleared before use (or replaced with a new empty table)? This was a potential problem with the previous implementation as well.

Reply:
Time to fix it!
Agree, the new empty table here is enough :)